api/handlers.go

```diff
@@ -435,6 +435,17 @@ func SearchHandler(c *gin.Context) {
 		}
 	}
 
+	//  Quick workround to allow Spanish support when the interface language is Spanish (AS-99).
+	if c.Query("language") == consts.LANG_SPANISH {
```

Comment:
Why do you need this part?
You have hasSpanish below?

Reply:
it is on a different file and package